### ISSUE TITLE
Multiples React packages don't work.

### ISSUE THREAD

**Issue by donavan-tangible** [exsurgo/directus-extension-react-example#2]
I'm not able to use some packages like:

import { useSearchParams } from 'react-router-dom';
import { useNavigate } from 'react-router-dom';

**Comment by exsurgo** [comment]
I don't think it will be possible to use React router within Directus, since it uses `vue-router` for routing.    If you really need to do routing within your extension, you might can try using it directly ... https://router.vuejs.org/
